8.cytopick_analysis/notebooks/find_prediction_images.ipynb

```diff
@@ -1,1914 +1,1911 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    """THIS PATH NEEDS TO BE CHANGED TO THE LOCAL IMAGE DIRECTORY ON YOUR MACHINE"""
</pre><p>Should this be a comment? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>